doc/ug/io.tex

```diff
@@ -1217,6 +1217,256 @@ \section{Error handling}
 continue, so you should really know what you do if you try and catch
 these errors.
 
+\section{Online-visualisation with Mayavi or OpenGL}
+\label{sec:visualizer}
+\index{Visualization}
+
+With the python interface, \es features two possibilities for
+online-visualization: 
+
+\begin{enumerate}
+\item Using the mlab module to drive \emph{Mayavi, a "3D scientific data
+visualization and plotting in Python"}. Mayavi has a user-friendly GUI to
+specify the appearance of the output. Additional requirements: XXXXXX.
+\item A direct rendering engine based on \emph{pyopengl}. As it is
+developed for \es, it supports the visualization of several specific features
+like external forces or constraints. It has no GUI to setup the appearance, but
+can be adjusted by a large set of parameters. Additional requirements: installed
+python module \emph{PyOpenGL}.  
+\end{enumerate}
+Both are not meant to produce high quality renderings, but rather to debug your
+setup and equilibration process.
+
+\subsection{General usage}
+
+The recommended usage of both tools is similar: Create the visualizer of your
+choice and pass it the \lstinline{espressomd.System()} object. Then write your
+integration loop in a seperate function, which is started in a non-blocking
+thread. Whenever needed, call \lstinline{update()} to synchronize the renderer
+with your system. Finally start the blocking visualization window with
+\lstinline{start()}. 
```

Comment:
Done. Mayavi has problems if there is only a single particle, so the minimal code now has two particles..